app/models/api_models.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel
+from models.coordinate import Coordinate
+class Prompt(BaseModel):
+    current_coodinate: tuple[float, float]     # type: Coordinate
+    prompt:  str    # type: str   ～したいなどの文字列
```

Comment:
`current_coordinate`に値をセットしないとエラーが出るということでしたが、モデルにNoneを許可することを明示しないと当たり前ですがバリデーションエラーが走ります. 
また`current_coordinate`は座標型を持っているはずなので以下のように書くと良いです
```suggestion
class Prompt(BaseModel):
    current_coodinate: Optional[Coordinate] = None
    prompt:  str    
```

Reply:
確かにこれは`current_coodinate: Optional[Coordinate] = None`って書いた方がわかりやすいな 🤔 

これでnullが来ても許容できるやん天才って思ったらこれでもエラー出た；；
まだなんか足りないのかな？
```
{
  "detail": [
    {
      "type": "float_type",
      "loc": [
        "body",
        "current_coodinate",
        "coordinate",
        0
      ],
      "msg": "Input should be a valid number",
      "input": null
    },
    {
      "type": "float_type",
      "loc": [
        "body",
        "current_coodinate",
        "coordinate",
        1
      ],
      "msg": "Input should be a valid number",
      "input": null
    }
  ]
}
```

Reply:
そういえば相談して解決したやつ書いてなかったからメモ
```
{
  "current_coordinate": null,
  "prompt": "string"
}
```
このように指定するとnullでもちゃんと走る
下記のURLのスキーマの部分
https://www.notion.so/cb53fd614251482b8649383b16d7205d